src/ROUTES.ts

```diff
@@ -780,6 +780,17 @@ const ROUTES = {
         route: 'settings/workspaces/:policyID/reportFields',
         getRoute: (policyID: string) => `settings/workspaces/${policyID}/reportFields` as const,
     },
+    // TODO: uncomment after development is done
+    // WORKSPACE_EXPENSIFY_CARD: {
+    //     route: 'settings/workspaces/:policyID/expensify-card',
+    //     getRoute: (policyID: string) => `settings/workspaces/${policyID}/expensify-card` as const,
+    // },
+    // WORKSPACE_EXPENSIFY_CARD_ISSUE_NEW: {
+    // route: 'settings/workspaces/:policyID/expensify-card/issues-new',
+    // getRoute: (policyID: string) => `settings/workspaces/${policyID}/expensify-card/issue-new` as const,
+    // },
+    // TODO: remove after development is done - this one is for testing purposes
+    WORKSPACE_EXPENSIFY_CARD_ISSUE_NEW: 'settings/workspaces/expensify-card/issue-new',
```

Comment:
Are these paths confirmed on the design doc?

Reply:
this one is for testing only - the ones above will be used when connected to each workspace setting page

Reply:
> this one is for testing only

Yeah, but I want to confirm the name of these paths

Reply:
Or we need to confirm the name of these paths on the next PRs that we use these paths

Reply:
Why do we need to confirm?

Reply:
it's all in the docs

Reply:
@DylanDylann Yeah these are coming from the docs so should be good to go

Reply:
Thanks. I am checking again